yjit/yjit.mk

```diff
@@ -20,28 +25,21 @@ yjit-static-lib-yes:
 	        -C overflow-checks=on \
 	        '--out-dir=$(CARGO_TARGET_DIR)/release/' \
 	        $(top_srcdir)/yjit/src/lib.rs
-
-yjit-static-lib-no:
+	touch $@
```

Comment:
It's probably worth leaving a comment above this line that explains why `touch` is necessary.

Reply:
Comments are not allowed in recipes.
If you write `#` there, it is run by `/bin/sh` (and ignored eventually).

Reply:
Moved to a macro, and commented there.